BenchTesting.java

```diff
@@ -62,5 +65,87 @@ public static void main(String[] args) {
+        System.out.println("Expected: island2(Sweetfish). Actual: " +ferry2.getSourceIsland());
+        System.out.println("Expected: island4(Egg). Actual: " +ferry3.getDestinationIsland());
+
+        System.out.println("Expected: true. Actual:\n " +ferry1.CanPassTravel(pass1));
+        System.out.println("Expected: false. Actual:\n " +ferry2.CanPassTravel(pass4));
+
+
+
+
+
+//        island1.enter(pass1);
+//        System.out.println("Expected: The person with the Pass can't travel to the next Island(luxury rate is lower). " +
+//                "Actual:\n "+ferry1.ProcessPass(pass1,island1,island2));
+        /////////////////////////////////////////////////////////////////////////////////
+//        Pass pass5 = new Pass(982, "Temp5", 10, 100, 1);
+//        Island island5 = new Island(444, "Pain", 2, 1);
+//        Ferry ferry4 = new Ferry(4,island4,island5);
+//
+//        island5.enter(pass1);
+//        System.out.println("Expected: The person with the Pass can't travel to the next Island ( exceed the maximum capacity of the destination island)." +
+//                "Actual:\n "+ferry1.ProcessPass(pass5,island4,island5));
+//////////////////////////////////////////////////////////////////////////////////
+//        Pass pass5 = new Pass(982, "Temp5", 10, 1, 1);
+//        Island island6 = new Island(444, "Pain", 2, 1);
+//        Island island7 = new Island(412, "Pain", 2, 2);
+//
+//        Ferry ferry5 = new Ferry(5,island6,island7);
+//
+//        island6.enter(pass1);
+//        System.out.println("Expected: The person with the Pass can't travel to the next Island ( The pass does not have enough credits for the ferry journey)." +
+//                "Actual:\n "+ferry5.ProcessPass(pass5,island6,island7));
+//////////////////////////////////////////////////////////////////////////////////////////
+//        Pass pass6 = new Pass(982, "Temp6", 10, 22, 1);
+//        Island island8 = new Island(422, "one", 2, 1);
+//        Island island7 = new Island(412, "two", 2, 2);
+//
+//        Ferry ferry6 = new Ferry(5,island7,island8);
+//
+//        island2.enter(pass1);
+//        System.out.println("Expected: The person with the Pass can't travel to the next Island ( the Pass is not listed in the source island for the ferry)." +
+//                " Actual:\n "+ferry6.ProcessPass(pass6,island7,island8));
+/////////////////////////////////////////////////////////////////////////////
+//        Pass pass7 = new Pass(982, "Temp7", 10, 22, 1);
+//        Island island9 = new Island(422, "one", 2, 1);
+//        Island island10 = new Island(412, "two", 2, 2);
+//
+//        Ferry ferry7 = new Ferry(5,island9,island10);
+//
+//        island9.enter(pass7);
+//        System.out.println("Expected: The person with the Pass can travel to the next Island ( Success )." +
+//                " Actual:\n "+ferry7.ProcessPass(pass7,island9,island10));
+
+        System.out.println(ferry1.toString());
```

Comment:
What's the expected value here?

Reply:
should  return a String representation of an object of that class including  and list of Passes

Reply:
Can you include that in the system.out.println?